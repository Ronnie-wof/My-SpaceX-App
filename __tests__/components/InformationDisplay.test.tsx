// tests the InformationDisplay component
import React from 'react';
import renderer from 'react-test-renderer';
import InformationDisplay from '../../src/components/InfomationDisplay';

describe('InformationDisplay component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<InformationDisplay label="Name" value="Ronnie" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
