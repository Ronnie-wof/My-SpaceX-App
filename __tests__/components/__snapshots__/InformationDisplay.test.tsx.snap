// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InformationDisplay component renders correctly 1`] = `
<View
  style={
    {
      "flexDirection": "row",
      "marginBottom": 5,
      "width": "100%",
    }
  }
>
  <Text
    style={
      {
        "fontSize": 18,
        "fontStyle": "italic",
        "fontWeight": "bold",
      }
    }
  >
    Name
    : 
  </Text>
  <View
    style={
      {
        "width": 600,
      }
    }
  >
    <Text
      style={
        {
          "fontSize": 18,
          "fontStyle": "italic",
        }
      }
    >
      Ronnie
    </Text>
  </View>
</View>
`;
